modules/client-configuration/pages/clients-opensuse.adoc

```diff
@@ -64,7 +79,7 @@ ifeval::[{uyuni-content} == true]
 
 |===
 
-include::snippets/add_channels_cli.adoc[]
+include::snippets/add_channels_scc.adoc[]
```

Comment:
We don't use SCC to get openSUSE on Uyuni.

Reply:
![image](https://user-images.githubusercontent.com/3914967/98321356-97122580-2030-11eb-9d67-984526cb67a3.png)